styles.css

```diff
@@ -53,7 +53,45 @@ h2 {
 .buttonContainer div a{
 	padding:30px;
 	text-decoration:none;
+	color:white;
+}
+
+.biography {
+	background-image: url("https://d2v9y0dukr6mq2.cloudfront.net/video/thumbnail/mCz7hqj/videoblocks-cogs-motion-background-loop-black-white_h85vxur9g_thumbnail-full01.png");
+	background-size:cover;
+	background-color:white;
+    background-position: center;
+	height:700px;
+	background-repeat: no-repeat;
+	overflow:auto;
+}
+
+@media only screen and (max-width:1180px) {
+	.biography .photo {
+		margin:0 auto;
+		float:none;}
 }
 
 
+.photo {
+	background-image: url("http://pcm.um.edu.my/wp-content/uploads/2017/11/empty-avatar-700x480.png");
+	height:500px;
+	width:500px;
+	color:white;
+	background-size:cover,contain;
+	background-repeat: no-repeat;
+	float:right;
+	margin-right:0px;}
 
+.about {
```

Comment:
this .about class needs a padding style - 10px or 15px, whatever you think looks good. Means that when the browser is smaller you can still read all the text